src/test/java/org/selenium/pages/tests/PageTest.java

```diff
@@ -0,0 +1,86 @@
+package org.selenium.pages.tests;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.selenium.ojb.Product;
+import org.selenium.utils.ListenerEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(ListenerEvent.class)
+public class PageTest extends BaseTest {
+    private List<Product> products = new ArrayList<>();
+    @BeforeEach
+    @Step("Beginning the action")
+    public void getStart() {
+        mainPage.logIn();
+        signIn.login();
+        account.goToMyWishList();
+    }
+
+    @Test
+    @Epic(value = "Cart")
+    @Description("Test description: Add product to car")
+    public void checkCartAdded() {
+        assertEquals(dresses.addToCart(3, products).toString(), cart.getCartList().toString());
+    }
+
+    @Test
+    @Epic(value = "Cart")
+    @Description("Test description: Comparing amount with the total ")
+    public void checkCartTotal() {
+        dresses.addToCart(3,products);
```

Comment:
How does the test work? I see from before method that you are on the wish list page. But this tests starts from dressed page